### PR TITLE
docs: document theme-mermaid in API sidebar

### PR DESCRIPTION
**Pull request by homotechsual**
## Pre-flight checklist

- [X] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Documentation for new theme introduced in 2.2.0.

### CODE REVIEW

**Comment by homotechsual** [comment]
I agree about the additional hint/wording not sure quite how to word it.

File: website/docs/api/themes/theme-mermaid.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+## Configuration {#configuration}
+
+```js title="docusaurus.config.js"
+module.exports = {
+  themes: ['@docusaurus/theme-mermaid'],
+  markdown: {
+    mermaid: true,
+  },
```
Comment (Josh-Cena) [inline website/docs/api/themes/theme-mermaid.md:21]: Let's just add a comment here.

```suggestion
  // In order for Mermaid code blocks in Markdown to work,
  // you also need to enable the Remark plugin with this option
  markdown: {
    mermaid: true,
  },
```

Comment (homotechsual) [reply website/docs/api/themes/theme-mermaid.md:23]: Sold :-)

**Review by Josh-Cena — APPROVED** [review]
LGTM

**Pull request merged**